tests/idv_service/session/retrieve/configuration/session.configuration.response.spec.js

```diff
@@ -0,0 +1,44 @@
+const CaptureResponse = require('../../../../../src/idv_service/session/retrieve/configuration/capture/capture.response');
+const SessionConfigurationResponse = require('../../../../../src/idv_service/session/retrieve/configuration/session.configuration.response');
+
+describe('SessionConfigurationResponse', () => {
+  let sessionConfigurationResponse;
+
+  beforeEach(() => {
+    sessionConfigurationResponse = new SessionConfigurationResponse({
+      client_session_token_ttl: 123,
+      session_id: 'some_id',
+      requested_checks: [
+        '',
```

Comment:
That seems bizarre... should we have a least 1 check, so then we can verify the length and the type of check it was converted to ?

Reply:
There is no conversion for the requested checks.

Reply:
ok